src/metabase/api/common/openapi.clj

```diff
@@ -86,21 +91,38 @@
     (->> (walk {:prefix ""} root)
          (filter #(-> % :route meta :schema)))))
 
+(defn- parse-compojure-bindings
```

Comment:
It'd be nice to see a unit test since I can't picture what args is supposed to be here, not a blocker tho.

Reply:
Oh, I should've done this, right, maybe that'll motivate me to support `[:as {{c :count} :query-params}]` syntax.

Reply:
It did 😁